utils/utils.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2024 OWASP ModSecurity Core Rule Set Project
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+func IsEscaped(input string, position int) bool {
```

Comment:
Do we want to add tests now that this one is exported?

Reply:
Yes. Will do.